client/ayon_core/hosts/substancepainter/plugins/load/load_mesh.py

```diff
@@ -54,12 +85,16 @@ def load(self, context, name, namespace, data):
             # only works for simple polygon scene
             result = prompt_new_file_with_mesh(mesh_filepath=path)
             if not result:
-                self.log.info("User cancelled new project prompt."
-                              "Creating new project directly from"
-                              " Substance Painter API Instead.")
-                settings = substance_painter.project.create(
-                    mesh_file_path=path, settings=sp_settings
-                )
+                if not substance_painter.project.is_open():
+                    self.log.info("User cancelled new project prompt."
+                                  "Creating new project directly from"
+                                  " Substance Painter API Instead.")
+                    settings = substance_painter.project.create(
+                        mesh_file_path=path, settings=sp_settings
+                    )
+                else:
+                    self.log.info("The project is already created after "
+                                  "the new project prompt action")
```

Comment:
How or when does this occur? This seems like we have an error in the logic elsewhere?